src/PKSim.UI/Views/ProteinExpression/TransferView.cs

```diff
@@ -133,58 +140,87 @@ public void SetData(DataTable transferTable, string selectedUnit)
             _selectedUnit = radioGroup.Properties.Items[radioGroup.SelectedIndex].Description;
          }
 
-         BindDataToGrid();
-         ConfigGridColumns();
+         bindDataToGrid();
       }
 
-      private void BindDataToGrid()
+      private void bindDataToGrid()
       {
          var transferTable = _transferData.Copy();
          transferTable.Columns.Add(
-            string.Concat(ColumnNamesOfTransferTable.RelativeExpressionOld.ToString(), STR_Percentage),
-            typeof(double), string.Concat(ColumnNamesOfTransferTable.RelativeExpressionOld.ToString(), "*100"));
+            RelativeExpressionOldPercentage,
+            typeof(double), string.Concat(RelativeExpressionOld, "*100"));
          transferTable.Columns.Add(
-            string.Concat(ColumnNamesOfTransferTable.RelativeExpressionNew.ToString(), STR_Percentage),
-            typeof(double), string.Concat(ColumnNamesOfTransferTable.RelativeExpressionNew.ToString(), "*100"));
+            RelativeExpressionNewPercentage,
+            typeof(double), string.Concat(RelativeExpressionNew, "*100"));
 
          var bindData = transferTable.DefaultView;
-         if (!String.IsNullOrEmpty(_selectedUnit))
-            bindData.RowFilter = String.Format("[{0}] = '{1}'", ColumnNamesOfTransferTable.Unit,
-                                              _selectedUnit);
+         if (!string.IsNullOrEmpty(_selectedUnit))
+            bindData.RowFilter = $"[{Unit}] = '{_selectedUnit}'";
+
          grdTransfer.BeginUpdate();
          grdTransfer.DataSource = null;
          grdTransfer.ResetBindings();
          grdTransfer.DataSource = bindData;
          grdTransfer.RefreshDataSource();
-         ConfigGridColumns();
+         configGridColumns(bindData);
          grdTransfer.EndUpdate();
       }
 
-      private void ConfigGridColumns()
+      private RepositoryItemBaseProgressBar initProgressBarEditor(int max)
+      {
+         return new RepositoryItemProgressBar
+         {
+            ReadOnly = true,
+            Enabled = false,
+            BorderStyle = BorderStyles.NoBorder,
+            Maximum = max,
+            Minimum = 0,
+            NullText = string.Empty,
+            PercentView = false,
+            ShowTitle = true,
+            DisplayFormat = {FormatType = FormatType.Numeric}
+         };
+      }
+
+      private int getPercentageMax(DataView dataView, string columnName)
```

Comment:
This is the change. Instead of setting the max to 100, I set the max to the max value in the column (int) and then this will be used as max for the repo. The scaling is then done automatically